x-pack/plugins/observability_solution/apm/public/components/app/service_tab_empty_state/constants.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export type EmptyStateKey =
+  | 'serviceOverview'
+  | 'serviceDependencies'
+  | 'infraOverview'
+  | 'serviceMap'
+  | 'transactionOverview'
+  | 'metrics'
+  | 'errorGroupOverview';
+
+interface EmptyStateContent {
+  title: string;
+  content: string;
+  imgName?: string;
+}
+
+export const emptyStateDefinitions: Record<EmptyStateKey, EmptyStateContent> = {
+  serviceOverview: {
+    title: i18n.translate('xpack.apm.serviceTabEmptyState.overviewTitle', {
+      defaultMessage: 'Detect and resolve issues faster with deep visibility into your application',
+    }),
+    content: i18n.translate('xpack.apm.serviceTabEmptyState.overviewContent', {
+      defaultMessage:
+        'Understanding your application performance, relationships and dependencies by instrumenting with APM.',
+    }),
+  },
+  serviceDependencies: {
+    title: i18n.translate('xpack.apm.serviceTabEmptyState.dependenciesTitle', {
+      defaultMessage: 'Understand the dependencies for your service',
+    }),
+    content: i18n.translate('xpack.apm.serviceTabEmptyState.dependenciesContent', {
+      defaultMessage:
+        'See your services dependencies on both internal and third-party services by instrumenting with APM.',
```

Comment:
is it `service's dependencies` ? 🤔

Reply:
@roshan-elastic 👆🏻

Reply:
Sorry yes. Can reword this a bit to avoid painful punctuation - technically it could be `service's` or `services'` but let's settle on `service's` given we're looking at an individual service:

'See your service's dependencies on both internal and third-party services by instrumenting with APM.'

Reply:
Since this PR is already merged I added the change in [this other PR](https://github.com/elastic/kibana/pull/192385) .